src/nft/css/sprinkles.css.ts

```diff
@@ -81,6 +81,7 @@ const dimensions = {
   '100': '100px',
   '120': '120px',
   '160': '160px',
+  '180': '180px',
```

Comment:
Since this is only used once, can you remove it from sprinkles and write it in as a custom style for width

Reply:
roger that